### ISSUE TITLE
Time-based axis

### ISSUE THREAD

**Issue by SPC-code** [KoalaPlot/koalaplot-core#64]
For dynamic data it is important to have time-based axis (preferably based on Kotlinx-datetime `Instant`. It requires #63 to work properly.

**Comment by gsteckman** [comment]
You can use a LongLinearAxisModel for representing date types and use kotlinx-datetime to format it. I have created a [sample](https://github.com/KoalaPlot/koalaplot-samples/blob/main/src/commonMain/kotlin/io/github/koalaplot/sample/TimeAxisLineChart.kt) that does this.

I realize this isn't the same as having a natively supported type, but I don't want to add kotlinx-datetime as a required dependency. This is a good use case for an extension library that people can optionally include to support additional data types.

**Comment by gsteckman** [comment]
Yes, I was considering creating a new repository for extensions, that would publish one or more artifacts for things like this that introduce additional dependencies.
